### PR TITLE
Switch back to cam stream once screen sharing is stopped.

### PR DESCRIPTION
**Pull request by linweigao**

### CODE REVIEW

File: src/components/Conference.tsx [hunk 1]
```diff
@@ -404,8 +405,8 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
 
         navigator.mediaDevices.getUserMedia(webcamScreenConstraints)
             .then(stream => {
-                this.localStream = stream;
-                this.gotStream(stream);
+                this.localCamStream = stream;
+                this.setLocalStream(stream);
```
Comment (niuyunsi) [inline src/components/Conference.tsx:409]: this.setLocalStream has not been implemented

File: src/components/Conference.tsx [hunk 2]
```diff
@@ -471,14 +479,15 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
                 let peerConnection = this.peerConnections[peerId];
                 if (oldStream) {
                     peerConnection.removeStream(oldStream);
+                    this.renegotiation[peerId] = true;
```
Comment (niuyunsi) [inline src/components/Conference.tsx:477]: Line 476-477 should be removed

Comment (linweigao) [reply src/components/Conference.tsx:477]: Good catch!

File: src/components/Conference.tsx (hunk 1 above)
Comment (linweigao) [reply src/components/Conference.tsx:409]: Good catch. Merge issue.

Comment (linweigao) [reply src/components/Conference.tsx:409]: done

File: src/components/Conference.tsx (hunk 2 above)
Comment (linweigao) [reply src/components/Conference.tsx:477]: done

**Review by niuyunsi — APPROVED** [review]

File: src/components/Conference.tsx [hunk 3]
```diff
@@ -429,33 +430,40 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
...
         const oldStream = this.state.localStream ? this.state.localStream.stream : null;
         this.setState({
             localStream: {
                 ...this.state.localStream,
                 ...conferenceStream,
                 stream,
                 local: true,
-                audioEnabled: true,
-                videoEnabled: true,
+                audioEnabled: this.state.localStream.audioEnabled,
```
Comment (asinner) [inline src/components/Conference.tsx]: I think you can remove these two since they are deconstructed above

Comment (linweigao) [reply src/components/Conference.tsx]: done

**Pull request merged**